templates/header.debian.html

```diff
@@ -5,12 +5,12 @@
 <p>
   This is the Debian package repository of {{product_name}} to automate installation and upgrade.
 
-  To use this repository, first add the key to your system:
+  To use this repository, first add the key to your system (for the Weekly Release Line):
 
   <pre class="text-white bg-dark">
     <code>
-  curl -fsSL <a href="{{web_url}}/{{organization}}-2023.key" style="color:white">{{web_url}}/{{organization}}-2023.key</a> | sudo tee \
```

Comment:
Is the equivalent of `curl -sS` not `wget -nv`?

Reply:
Indeed, just keep it in sync with the Fedora `wget` instruction [here](https://www.jenkins.io/doc/book/installing/linux/#fedora). I can change it, in case you want it, but in this case we should also adjust the Fedora part.

Reply:
+1 for being consistent with other pages.